packages/siwe/lib/types.ts

```diff
@@ -23,9 +23,12 @@ export interface VerifyOpts {
 
     /** If the library should reject promises on errors, defaults to false */
     suppressExceptions?: boolean;
+
+    /** Function to be called when the message cannot be validated. */
```

Comment:
I think this should be reworded, as it is signature verification rather than message validation.